src/bindings/python/src/compatibility/openvino/__init__.py

```diff
@@ -2,4 +2,72 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore # mypy issue #1422
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+try:
+    # Internal modules
+    from openvino import test_utils as test_utils
+except ImportError:
+    pass
+
+try:
+    # Import all public modules
+    from openvino import runtime as runtime
+    from openvino import frontend as frontend
+    from openvino import helpers as helpers
+    from openvino import preprocess as preprocess
+    from openvino import utils as utils
+    from openvino.runtime import properties as properties
+    # Import old API
+    # TODO: remove in 2024.0
+    from openvino import inference_engine as inference_engine
+
+    # Required for Windows OS platforms
+    from openvino.utils import _add_openvino_libs_to_search_path
+
+    _add_openvino_libs_to_search_path()
+
+    # Import most important classes and functions from openvino.runtime
+    from openvino.runtime import Model
+    from openvino.runtime import Core
+    from openvino.runtime import CompiledModel
+    from openvino.runtime import InferRequest
+    from openvino.runtime import AsyncInferQueue
+
+    from openvino.runtime.op import Constant
+    from openvino.runtime.op import Parameter
+
+    from openvino.runtime import Extension
+    from openvino.runtime import Dimension
+    from openvino.runtime import Strides
+    from openvino.runtime import PartialShape
```

Comment:
how about `layout_helpers`. I found it is used quite often.

Reply:
Added